apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -28,40 +38,61 @@ const DepartureList = ({
   );
 
   let departures: DepartureInfo[] = [];
+  const groupedAlerts = alertsByRoute(alerts);
+  const alertsForRoute = groupedAlerts[route.id] || [];
 
+  const alertsByDirectionObj = alertsByDirectionId(alertsForRoute);
+  const alertsAffectingBothDirectionsArray = alertsAffectingBothDirections(
+    alertsForRoute
+  );
+
+  const alertsDirectionArray = alertsByDirectionObj[directionId]
+    ? alertsByDirectionObj[directionId]
+    : [];
+
+  const allAlerts = concat(
+    alertsAffectingBothDirectionsArray,
+    alertsDirectionArray
+  ).filter(alert => {
+    return ["detour", "suspension", "shuttle"].includes(alert.effect);
+  });
   // TODO: handle no predictions or schedules case and predictions only case
   return (
     <>
-      {schedules.length && (
-        <div className="stop-departures departure-list-header">
-          <div className={`departure-card__route ${routeBgClass(route)}`}>
-            <div>
-              {renderSvg("c-svg__icon", routeToModeIcon(route), true)}{" "}
-              {routeName(route)}
+      <Alerts alerts={allAlerts} />
```

Comment:
question: Can there be more than one alert here? Does the order matter?

Reply:
There can be more than one alert based on the `concat` ordering the route level alerts will appear first than stop level.